tests/01_03_06-2wp_no_locking_cap.js

```diff
@@ -1,116 +1,79 @@
 const expect = require('chai').expect
-var { sequentialPromise, wait } = require('../lib/utils');
-const peglib = require('peglib');
-const bitcoin = peglib.bitcoin;
-const rsk = peglib.rsk;
-const pegUtils = peglib.pegUtils;
-const pegAssertions = require('../lib/assertions/2wp');
-const whitelistingAssertionsLegacy = require('../lib/assertions/whitelisting-legacy');
-const CustomError = require('../lib/CustomError');
 const rskUtils = require('../lib/rsk-utils');
 const { getRskTransactionHelpers } = require('../lib/rsk-tx-helper-provider');
 const { getBtcClient } = require('../lib/btc-client-provider');
+const { sendPegin, ensurePeginIsRegistered, sendTxToBridge, MINIMUM_PEGIN_VALUE_IN_BTC } = require('../lib/2wp-utils');
+const { getBridge, getLatestActiveForkName } = require('../lib/precompiled-abi-forks-util');
+const { getDerivedRSKAddressInformation } = require('@rsksmart/btc-rsk-derivation');
+const btcEthUnitConverter = require('btc-eth-unit-converter');
+const whitelistingAssertions = require('../lib/assertions/whitelisting');
 
 describe('Transfer BTC to RBTC before papyrus200', function() {
 
-  var federationAddress;
-  var btcClient;
-  var rskClient;
-  var pegClient;
-  var test;
-  let whitelistTestLegacy;
   let rskTxHelpers;
   let btcTxHelper;
+  let rskTxHelper;
 
-  const NETWORK = bitcoin.networks.testnet;
+  const MINIMUM_PEGIN_VALUE_IN_SATOSHI = btcEthUnitConverter.btcToSatoshis(MINIMUM_PEGIN_VALUE_IN_BTC)
+  const MINIMUM_PEGIN_VALUE_IN_WEIS = btcEthUnitConverter.btcToWeis(MINIMUM_PEGIN_VALUE_IN_BTC)
```

Comment:
```suggestion
  const PEGOUT_VALUE_IN_WEIS = btcEthUnitConverter.btcToWeis(MINIMUM_PEGIN_VALUE_IN_BTC)
```
Makes more sense right? You are using that value for peg-outs not peg-ins

Reply:
hmm, for the peg-out i'm actually using the BTC value
`await sendTxToBridge(rskTxHelper, MINIMUM_PEGIN_VALUE_IN_BTC, recipientRskAddressInfo.address);`

the value converted to WEIS i'm only using when asserting the recipientRSKAddress balance, because the balance returned is in WEIS...

Reply:
discussed on the call -> adding new constants